frontend/public/components/storage/create-pvc.tsx

```diff
@@ -80,14 +81,20 @@ export class CreatePVCForm extends React.Component<CreatePVCFormProps, CreatePVC
     this.setState({ nameValuePairs }, this.onChange);
   };
 
+  handleCephProvisioner = (sc: string) => {
+    return isCephProvisioner(sc)
+      ? ['ReadWriteOnce', 'ReadWriteMany']
+      : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+  };
+
   handleStorageClass = (storageClass) => {
     //if the provisioner is unknown or no storage class selected, user should be able to set any access mode
     const modes =
       storageClass &&
       storageClass.provisioner &&
       provisionerAccessModeMapping[storageClass.provisioner]
         ? provisionerAccessModeMapping[storageClass.provisioner]
-        : ['ReadWriteOnce', 'ReadWriteMany', 'ReadOnlyMany'];
+        : this.handleCephProvisioner(storageClass.provisioner);
     //setting message to display for various modes when a storage class of a know provisioner is selected
     const displayMessage =
       storageClass && provisionerAccessModeMapping[storageClass.provisioner]
```

Comment:
Note that the message won't update for Ceph provisioners. Is that intended?

Reply:
Yes, as per discussion with the UX team, the message should be same.

Reply:
> Yes, as per discussion with the UX team, the message should be same.

But it's *not* the same as other provisioners unless I'm misreading the code.

Reply:
ack, have updated it.